src/operators.jl

```diff
@@ -2,7 +2,7 @@
     complement(g)
 
 Return the [graph complement](https://en.wikipedia.org/wiki/Complement_graph)
-of a graph
```

Comment:
The functions in `operators.jl` should work on `AbstractGraph`, with specializations as overrides in other graph packages.